src/fetch_git.c

```diff
@@ -30,6 +30,11 @@
 #include "fetch.h"
 #include "fetch_git.h"
 
+// The main branch ("main") is fetched by default, but we also support to
+// fetch the legacy one (i.e. "master") because it is required by some old
+// repositories.
+static const gchar *g_git_branches[] = {"main", "master"};
```

Comment:
You can still use a define here if you wish..

static const gchar *g_git_branches[] = { GIT_BRANCHES };

Reply:
Done. BTW, I moved the comment from `fetch_git.c` to `fetch_git.h`, @p3ck,  please help to take a look again, thanks!